src/main/java/dev/efnilite/vilib/inventory/Menu.java

```diff
@@ -20,6 +21,10 @@
 import org.jetbrains.annotations.Nullable;
 
 import java.util.*;
+import java.util.concurrent.CompletableFuture;
```

Comment:
These imports aren't used anywhere in the rest of this commit

Reply:
Those are outdated and were basically never there, that was from a previous commit that hasn't been changed.

Reply:
When I edit them it doesn't show the import is there.

Reply:
That's fine.